src/components/SideMenu.tsx

```diff
@@ -80,7 +83,7 @@ const SideMenu = () => {
           </Link>
         </li>
       </SideMenuNav>
-      <SignOutButton>Sign Out</SignOutButton>
+      <SignOutButton onClick={() => actions.signOut()}>Sign Out</SignOutButton>
```

Comment:
hooks up sign out. this will remove the browser cookie, and causes the ui to jump back to the login screen